Update PHP APM configuration documentation for version 0.76.0 (#14447)
DataDog/documentation: content/en/tracing/setup_overview/setup/php.md

**Default**: `null`<br>
Set an application’s version in traces and logs, for example: `1.2.3`, `6c44da20`, `2020.02.13`. Added in version `0.47.0`.

`DD_TRACE_HTTP_URL_QUERY_PARAM_ALLOWED`
: **INI**: `datadog.trace.http_url_query_param_allowed`<br>
**Default**: `*`<br>

> A comma-separated list of query parameters to be collected as part of the URL, this can also be empty to prevent collecting any parameters or `*` to collect all parameters. Added in version `0.74.0`.

Reviewer: ```suggestion
A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
```

**Default**: `null`<br>
Set an application’s version in traces and logs, for example: `1.2.3`, `6c44da20`, `2020.02.13`. Added in version `0.47.0`.

`DD_TRACE_HTTP_URL_QUERY_PARAM_ALLOWED`
: **INI**: `datadog.trace.http_url_query_param_allowed`<br>
**Default**: `*`<br>
A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.

> `DD_TRACE_CLIENT_IP_HEADER_DISABLED`

Reviewer 2: We tend to use the `_ENABLED` pattern rather than the `_DISABLED` pattern. E.g. `DD_TRACE_ENABLED` and `DD_TRACE_CLI_ENABLED`, in this case with a default value of 1. Would you agree with this change?

Reviewer 3: I think "disabled" make sense in this case, because the ip collection is enabled by default. So if the user is looking for a config value relating to ip collection, it's probably because they decided they want to disable this setting.

Reviewer 2: I am fine with proceeding with "disabled", especially since this has already been discussed and implemented cross team.
However, I would recommend for future params to use the "enable" pattern, not only for consistency, but because we learned from past experience with users that the double negation to enable something ("disabled" + "false/0") causes confusion.
